setup.cfg

```diff
@@ -20,32 +20,32 @@ package_dir =
 packages = find:
 python_requires = >=3.7
 install_requires = 
-    pyjwt >= 2
-    cryptography >= 3
-    jsonschema >= 4
-    boto3 >= 1.24.66
-    Pillow >= 9
-    requests >= 2
-    shortuuid >= 1
-    pyIIIFpres >= 0.1.1
-    python-magic >= 0.4.24
-    Deprecated >= 1
-    backports.zoneinfo;python_version<"3.9"
+    pyjwt ~= 2.4
+    cryptography ~= 3.3
+    jsonschema ~= 4.15
+    boto3 ~= 1.26
+    Pillow ~= 9.3.0
```

Comment:
I think this is already equivalent to `Pillow ~= 9.3`, right? I think we'd only need to specify a minimum patch version if it's greater than 0.